src/core/manager.c

```
@@ -2610,23 +2610,23 @@ int manager_deserialize(Manager *m, FILE *f, FDSet *fds) {
                         uint32_t id;
 
                         if (safe_atou32(val, &id) < 0)
-                                log_debug("Failed to parse current job id value %s", val);
```

hmm, not sure how I feel about this. The original intention was that errors like this are semi-expected during version updates, and not a problem nor actionable.

---

Well, we expect to remove stuff occasionally, but I don't think we ever mean to make old serialization formats unparseable on purpose. Are you aware of any instances of that happening? I'm aware of "kdbus-fd=" going away, and we explicitly silently ignore that.

---

I agree with @keszybz . It appears that a deserialisation error indicates a bug, not some expected event.

By the way, I spent two hours trying to find out why `daemon-reexec` had frozen `systemd`. It would have helped a lot if I had seen something like `Failed to parse environment entry`.